pkg/worker/runc_server.go

```diff
@@ -111,6 +111,8 @@ func (s *RunCServer) RunCExec(ctx context.Context, in *pb.RunCExecRequest) (*pb.
 	if !exists {
 		return &pb.RunCExecResponse{Ok: false}, nil
 	}
+	process.Env = append(process.Env, instance.Spec.Process.Env...)
```

Comment:
I added this env here as well. It is not necessary, but I thought it was strange that env included using `with_env` or `env_vars` was not available.